srtcore/srt.h

```diff
@@ -317,8 +328,8 @@ enum CodeMinor
     MN_NOLISTEN = 6,
     MN_ISRENDEZVOUS = 7,
     MN_ISRENDUNBOUND = 8,
-    MN_ISSTREAM = 9,
-    MN_ISDGRAM = 10,
+    MN_INVALMSGAPI = 9,
+    MN_INVALBUFFERAPI = 10,
```

Comment:
That's an API break, can't you just deprecate the ISSTREAM/ISDGRAM and put the new ones at new numbers ?

Reply:
The problem is that these names refer to something that is a kinda "removed feature". It's a UDT legacy, but never existed in SRT.

Maybe you're right, the old numbers better remain unchanged, just marked as deprecated, and the new ones should be in use. This would be later recycled after some version of deprecation.